master/internal/experiment.go

```diff
@@ -373,6 +376,23 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 		}
 
 		ctx.Log().Info("experiment shut down successfully")
+
+	case *apiv1.ActivateExperimentRequest:
+		switch ok := e.updateState(ctx, model.ActiveState); ok {
+		case true:
+			ctx.Respond(&apiv1.ActivateExperimentResponse{})
+		default:
+			ctx.Respond(status.Errorf(codes.FailedPrecondition,
+				"experiment in incompatible state %s", e.State))
+		}
+	case *apiv1.PauseExperimentRequest:
+		switch ok := e.updateState(ctx, model.PausedState); ok {
```

Comment:
non-blocking: this is better as just an if-else given that this is just a bool variable.